cve_bin_tool/checkers/sqlite.py

```diff
@@ -10,10 +10,10 @@
 
 """
 try:
-    import urllib.request as request
+    from urllib.request import urlopen
```

Comment:
Redundant change. 
Importing `urllib.request` or from `urllib.request import urlopen` imports the entire module so there is no difference between them.

Reply:
https://softwareengineering.stackexchange.com/questions/187403/import-module-vs-from-module-import-function